contracts/LifMarketMaker.sol

```diff
@@ -153,28 +153,18 @@ contract LifMarketMaker is Ownable {
     assert(totalPeriods == 24 || totalPeriods == 48);
     require(startBlock >= block.number);
     require(blocksPerPeriod > 0);
+    require(marketMakerPeriods[i].startBlock > 0);
```

Comment:
`i` is probably not defined here. Anyways, why not just `require(marketMakerPeriods.length > 0)`?